src/Commands/StartRoadRunnerCommand.php

```diff
@@ -144,9 +144,9 @@ protected function configPath()
         $path = $this->option('rr-config');
 
         if (! $path) {
```

Comment:
By introducing a default value for the `--rr-config` option, this part is never called anymore. The default `.rr.yaml` is now also not wrapped with `base_path()` anymore, so the check on L152 should actually fail for fresh installs.

Since your problem seems to be container related, why not simply add `RUN touch /var/www/.rr.yaml` in your `Dockerfile` to match the fallback behavior on L147 without granting too many permissions?

Reply:
But if the file is not specified, then when running the `php artisan octane:start` code updates the access time via `touch()`, and the file is not allowed to be changed